### ISSUE TITLE
Image load from exist object

### ISSUE THREAD

**Issue by seapy** [nakajijapan/PhotoSlider#18]
Reuse image not remote url when application already has a image object.

like this

``` swift
var myImage = UIImage(named: 'wow')
var slider = PhotoSlider.ViewController(images: [myImage])
```

**Comment by nakajijapan** [comment]
ok!

**Comment by nakajijapan** [comment]
Hi, @seapy.

I just fixed this issue, please try to run in v0.7.0.

**Comment by seapy** [comment]
:+1: Thanks ~
